package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@dsnp/activity-content",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LibertyDSNP/activity-content.git"
+  },
+  "license": "ISC",
+  "scripts": {
+    "build:cjs": "tsc -p tsconfig.json",
+    "ts-types": "tsc --emitDeclarationOnly --outDir ./dist/types",
+    "build": "mkdir -p ./dist/types/ && esbuild --format=cjs ./src/index.ts src/core/index.ts --bundle --outdir=./dist --outbase=src && npm run ts-types",
+    "clean": "rm -Rf dist ./*.d.ts ./*.js ./*.tgz",
+    "doc": "rm -Rf dist/docs && npm run doc:markdown",
+    "doc:html": "typedoc --options ./typedoc.json --plugin none --theme default --out dist/docs",
+    "doc:config": "typedoc --showConfig --options ./typedoc.json --plugin none --theme default --out dist/docs",
+    "doc:json": "typedoc --options ./typedoc.json --json dist/docs/typedoc.json",
+    "doc:markdown": "typedoc --options ./typedoc.json --theme markdown --out dist/docs",
+    "format": "eslint \"**/*.{js,ts}\" --fix",
+    "lint": "eslint \"**/*.{js,ts}\"",
+    "pre-publish": "cp package.json dist",
+    "pub": "cd dist & npm publish",
+    "postpub": "cd dist & del package.json",
+    "pack": "npm run pre-publish && cd dist && npm pack && mv *.tgz ../",
+    "test": "jest --runInBand"
+  },
+  "author": "",
+  "bugs": {
+    "url": "https://github.com/LibertyDSNP/activity-content/issues"
+  },
+  "homepage": "https://github.com/LibertyDSNP/activity-content#readme",
+  "dependencies": {
+    "esbuild": "^0.13.13",
+    "js-sha3": "^0.8.0"
+  },
+  "jest": {
```

Comment:
● Multiple configurations found:
    * /Users/wil/liberty/activity-content/jest.config.ts
    * `jest` key in /Users/wil/liberty/activity-content/package.json

  Implicit config resolution does not allow multiple configuration files.
  Either remove unused config files or select one explicitly with `--config`.

  Configuration Documentation:
  https://jestjs.io/docs/configuration.html